README.md

```diff
@@ -16,25 +16,39 @@ Or
 
 
 ## Run with Docker
-
+- If you are on your local machine, we recommend that you run this container using Docker.  
+- We have tested running this container using Docker on local mac machines and also on a Pawsey Nimbus instance.
```

Comment:
Whats the relevant specs of a pawsey nimbus instance?

Reply:
We have tested this container using Docker on local mac machines and also on the Pawsey Nimbus cloud instance `Pawsey Bio - Ubuntu 22.04 - 2023-06`